assets/css/views/images.css

```diff
@@ -0,0 +1,317 @@
+  color: var(--image-show-link-hover-color);
+}
+
+/* This and the one above needed for older browsers for some reason, appears to be a bug */
+.image-target {
+  max-width: 100%;
+}
+
+/* Resizes large images to fit in a smaller container; width and height are defined inline */
+.image-constrained {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.image-constrained img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.image-scaled {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+/* Due to the address bar hiding/reappearing in mobile browsers, viewport
+ * units are only reliable in desktop browsers. */
+@media (min-width: $min-desktop-width) {
+  .image-scaled {
+    max-height: 80vh;
+  }
+}
+
+.image-partscaled {
+  max-width: 100%;
+}
+
+.image-description {
+  overflow: auto;
+}
+
+.image-description img {
+  max-width: 100%;
+}
+
+.image-size {
+  white-space: nowrap;
+}
+
+.js-source-link {
+  word-wrap: break-word;
+}
+
+div.image-container img,
+div.image-container video {
+  vertical-align: middle;
+  max-width: 100%;
+  max-height: 100%;
+}
+
+#image-source > p {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.interaction-user-list-item {
+  display: inline-block;
+  padding: 2px;
+  padding-right: 7px;
+}
+
+.image_menu {
+  padding-left: 0;
+  margin: 1em 0 0;
+}
+
+.horizontal-list {
+  margin: 0;
+  padding-left: 0;
+  list-style: none;
+}
+
+.horizontal-list li {
+  display: inline-block;
+}
+
+.horizontal-list li + li {
+  margin-left: 1em;
+}
+
+.dnp-warning {
+  padding: 0.5em;
+  border: 3px solid var(--vote-down-color);
+  background: var(--admin-links-color);
+}
+
+.dnp-warning h4 {
+  margin-top: 0.3em;
+  margin-bottom: 0.2em;
+}
+
+.derpy__2fa {
+  width: 25%;
+  min-width: 200px;
+}
+
+.derpy__login {
+  margin-left: 8px;
+  align-self: flex-end;
+}
+
+#js-image-upload-previews {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+#js-image-upload-previews:empty {
+  display: none;
+}
+
+#js-image-upload-previews img,
+#js-image-upload-previews video {
+  max-height: 200px;
+  max-width: 200px;
+}
+
+#js-image-upload-previews .scraper-preview--label {
+  cursor: pointer;
+  margin: 0 8px 8px 0;
+}
+
+#js-image-upload-previews .scraper-preview--label .scraper-preview--image-wrapper {
+  display: inline-block;
+  padding: 9px;
+  border: 1px solid transparent;
+  opacity: 0.6;
+}
+
+#js-image-upload-previews .scraper-preview--label .scraper-preview--image {
+  background: url("data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeD0iMHB4IiB5PSIwcHgiIHZpZXdCb3g9IjAgMCAxNiAxNiIgZW5hYmxlLWJhY2tncm91bmQ9Im5ldyAwIDAgMTYgMTYiIHhtbDpzcGFjZT0icHJlc2VydmUiPjxyZWN0IGZpbGw9IiNGRkZGRkYiIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIvPjxyZWN0IGZpbGw9IiNFRkVGRUYiIHdpZHRoPSI4IiBoZWlnaHQ9IjgiLz48cmVjdCBmaWxsPSIjRUZFRkVGIiB3aWR0aD0iOCIgaGVpZ2h0PSI4IiB4PSI4IiB5PSI4Ii8+PC9zdmc+Cg==")
```

Comment:
To followup: this appears to just be the checkerboard image (mark this as resolved)